pkgs/games/natural-pong/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, udev, alsa-lib, vulkan-loader, xorg, libxkbcommon, wayland, pkg-config, makeWrapper, rustPlatform, fetchFromGitHub }:
+let
+  buildInputs = [
+    udev alsa-lib vulkan-loader
+    xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi
+    libxkbcommon wayland
+  ];
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+in rustPlatform.buildRustPackage {
+  pname = "natural-pong";
+  version = "0.1.0";
+  cargoHash = "sha256-jfeQjZrGyGXQ298naakiDpSlbl116ZKyi0RBkLnlwsA=";
+
+  buildInputs = buildInputs;
+  nativeBuildInputs = nativeBuildInputs;
```

instead of using the let binding you should just do:
```suggestion
  buildInputs = [
    udev 
    alsa-lib 
    vulkan-loader
    xorg.libX11 
    xorg.libXcursor 
    xorg.libXrandr
    xorg.libXi
    libxkbcommon
    wayland
  ];
  nativeBuildInputs = [
    pkg-config
    makeWrapper
  ];
```